letsencrypt-auto-source/letsencrypt-auto.template

```diff
@@ -720,6 +764,12 @@ UNLIKELY_EOF
       error "WARNING: unable to check for updates."
     fi
 
+    # If for any reason REMOTE_VERSION is not set, let's assume certbot-auto is up-to-date,
+    # in order to avoid potential failures because of an unset REMOTE_VERSION variable.
+    if [ -z "$REMOTE_VERSION" ]; then
+      REMOTE_VERSION="$LE_AUTO_VERSION"
+    fi
+
     LE_VERSION_STATE=`CompareVersions "$LE_PYTHON" "$LE_AUTO_VERSION" "$REMOTE_VERSION"`
```

Comment:
I did that.

Reply:
In fact I rewrote your idea into this: if `REMOTE_VERSION` is empty, skip the entire self-upgrading process. It is functionally equivalent, since the self-upgrading process is invoked only for `REMOTE_VERSION` values `UNOFFICIAL` or `OUTDATED`. But it is less error-prone because here now is explicit that the process is skipped, since the code is in a if/else dedicated branch.